shopfloor_reception/tests/test_multi_barcode.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+import mock
+
+from odoo import fields
+
+from odoo.addons.shopfloor.actions.barcode_parser import BarcodeParser, BarcodeResult
+
+from .common import CommonCase
+
+
+class TestMultiBarcode(CommonCase):
```

Comment:
To remove confusion with multi barcode module
```suggestion
class TestStructuredBarcode(CommonCase):
```

Reply:
@jbaudoux Done